plane_spotter/geolocator.py

```diff
@@ -1,15 +1,87 @@
 from typing import Any
 
+import csv
+import haversine
+
+AIRPORT_TYPE_BLACKLIST = set(('balloonport', 'closed', 'heliport', 'seaplane_base'))
+
 class Geolocator:
   ''' Translates coordinates to the closest airport. '''
 
   def __init__(self, airport_code_file: str='plane_spotter/data/airport-codes.csv') -> 'Geolocator':
     ''' Initializes a Geolocator object '''
-    pass
+    self.__airports = []
+    # Add airport dict for each row in airport_code_file
+    with open(airport_code_file, 'r') as file:
+      reader = csv.reader(file)
+      headers = next(reader)
+      for line in reader:
+        airport = {}
+        # Convert row to dict
+        for key, value in zip(headers, line):
+          airport[key] = value
+        # Add it if it's not a blacklisted type
+        if airport['type'] not in AIRPORT_TYPE_BLACKLIST:
+          coordinates = airport['coordinates'].split(', ')
+          airport['coordinates'] = (float(coordinates[0]), float(coordinates[1]))
+          self.__airports.append(airport)
+    if len(self.__airports) == 0:
+      raise Exception('No airports loaded.')
```

Comment:
Maybe `RuntimeError` is more appropriate

Reply:
Good point!